pol-core/pol/network/auxclient.cpp

```diff
@@ -234,6 +237,9 @@ void AuxClientThread::run()
     std::this_thread::sleep_for( std::chrono::seconds( 1 ) );
 
   Core::PolLock lock;
+  // hold also the transmit mutex, the counter syncs but not in a way that the threadsanitizer
+  // recognizes is.
+  std::unique_lock<std::mutex> transmitlock( _transmit_mutex );
```

Comment:
is there any situation where the transmitlock might be used before the pollock?

Reply:
It's never used with PolLock, the actual transmit where the lock is needed runs without our global lock